packages/vc-export/src/KiltAttestationProofV1.ts

```diff
@@ -577,7 +580,12 @@ export function initializeProof(
   KiltAttestationProofV1,
   Parameters<ApiPromise['tx']['attestation']['add']>
 ] {
-  const { credentialSubject, credentialSchema } = credential
+  const { credentialSubject, credentialSchema, nonTransferable } = credential
+
+  if (nonTransferable !== true) {
```

Comment:
it's not - your suggestion checks for 'falsiness', so that `null`, `""`, or `0` would trigger this branch, while `[]`, `-1`, etc. would not. The point here is: anything but `true` triggers this branch.